enterprise/frontend/src/metabase-enterprise/settings/selectors.ts

```diff
@@ -19,5 +19,9 @@ export const getLoadingMessage = (state: EnterpriseState) =>
   LOADING_MESSAGE_BY_SETTING[getSetting(state, "loading-message")];
 
 const DEFAULT_APPLICATION_NAME = "Metabase";
-export const getIsWhiteLabeling = (states: EnterpriseState) =>
-  getSetting(states, "application-name") !== DEFAULT_APPLICATION_NAME;
+export const getIsWhiteLabeling = (state: EnterpriseState) =>
+  getApplicationName(state) !== DEFAULT_APPLICATION_NAME;
+
+export function getApplicationName(state: EnterpriseState) {
```

Comment:
We're going to use this selector to replace the application name deliberately rather than relying on `ttag` hack.